utils_cv/detection/plot.py

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
```

Comment:
Same comment as before: if you copied (and edited) this code from somewhere, can you please provide a link.

Reply:
wrote these funcs